### PR TITLE
Fix inspect output for falsey values

### PR DESCRIPTION
**Pull request by zokioki**
This change ensures that falsey values are reflected correctly when printing
out the final result - they currently print out as `nil`. For example:

```ruby
class MyStoreModel < StoreModel
  attribute :one, :integer, default: 1
  attribute :two, :boolean, default: false
  attribute :three, :string, default: nil
end
```

Current Behavior: `<MyStoreModel one: 1, two: nil, three: nil>`
Expected Behavior:  `<MyStoreModel one: 1, two: false, three: nil>`

### CODE REVIEW

**Comment by DmitryTsepelev** [comment]
Hi @zokioki, thanks for the PR, it looks great! Could you please add a test for that (to make sure we won't have a regression in future) and add yourself into the changelog?

**Comment by zokioki** [comment]
@DmitryTsepelev Thanks for a useful library! Just updated the specs to reflect nil and boolean attributes, updated the changelog and bumped the version as well. Let me know if that looks good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,5 +61,5 @@
 
 - Initial version ([@DmitryTsepelev][])
 
-[@DmitryTsepelev]: https://github.com/DmitryTsepelev
+[@dmitrytsepelev]: https://github.com/DmitryTsepelev
```
Comment (DmitryTsepelev) [inline CHANGELOG.md]: please switch this line back

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.5.4 (2019-09-17)
```
Comment (DmitryTsepelev) [inline CHANGELOG.md]: I'll bump the version by myself so let's remove this line

File: lib/store_model/version.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module StoreModel # :nodoc:
-  VERSION = "0.5.3"
```
Comment (DmitryTsepelev) [inline lib/store_model/version.rb]: please revert, I'll bump version by myself

File: CHANGELOG.md (hunk 1 above)
Comment (zokioki) [reply CHANGELOG.md]: weird, looks like one of my linters tweaked this - reverted via https://github.com/DmitryTsepelev/store_model/pull/38/commits/a8d9cb3d437b6aedbbd88c6027227d9885c46ad0.

**Pull request merged**